presto-session-property-managers/src/main/java/io/prestosql/plugin/session/SessionMatchSpec.java

```diff
@@ -120,4 +127,43 @@ public SessionMatchSpec(
     {
         return sessionProperties;
     }
+
+    public static class Mapper
+            implements RowMapper<SessionMatchSpec>
+    {
+        @Override
+        public SessionMatchSpec map(ResultSet resultSet, StatementContext ctx) throws SQLException
+        {
+            Map<String, String> sessionProperties = getProperties(Optional.ofNullable(resultSet.getString("session_property_names")),
```

Comment:
Nit: this would be easier to read on two lines
```java
Map<String, String> sessionProperties = getProperties(
        Optional.ofNullable(resultSet.getString("session_property_names")),
        Optional.ofNullable(resultSet.getString("session_property_values")));